packages/react-openapi/src/OpenAPISchema.tsx

```diff
@@ -14,6 +14,7 @@ interface OpenAPISchemaPropertyEntry {
     propertyName?: string;
     required?: boolean;
     schema: OpenAPIV3.SchemaObject;
+    example?: string;
```

Comment:
I don't understand why we need this prop. Isn't `example` included in the `schema` prop (https://github.com/kogosoftwarellc/open-api/blob/main/packages/openapi-types/index.ts#L166) ?

Reply:
Ok it looks like `example` is for parameters and `examples` is for schema fields. I think we should try to handle both the same way.

Reply:
It looks like you don't actually use this

```suggestion
```

and you use `schema.example`.

Reply:
@SamyPesse it looks like you got it reversed. `example` is for schema fields, and `examples` is for parameters
<img width="1114" alt="Screen Shot 2024-05-21 at 2 56 09 PM" src="https://github.com/GitbookIO/gitbook/assets/140522517/d9e6c146-66a2-44c4-b222-da44becbbc85">

https://swagger.io/docs/specification/adding-examples/

Reply:
I also think we should keep this PR just for the singular `example` and not support `examples` for now. What do you think?

Reply:
Yes, but it doesn't change the fact that this prop is actually not used:

```suggestion
```

Reply:
removed